interfaces/Portalicious/src/app/services/log.service.ts

```diff
@@ -0,0 +1,113 @@
+import { Injectable } from '@angular/core';
+import { DebugPlugin } from '@microsoft/applicationinsights-debugplugin-js';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { environment } from 'src/environments/environment';
+
+// All properties sorted alphabetically.
+export enum LogEvent {
+  userLogout = 'user/logout',
+}
+
+@Injectable({
+  providedIn: 'root',
+})
+export class LogService {
+  private appInsights: ApplicationInsights;
+  private appInsightsEnabled: boolean;
```

Comment:
Maybe more indicative of what this represents:

```suggestion
  private appInsightsInitialized: boolean;
```

Reply:
Ah, yes.. This was a remnant of a time where there where multiple "logging services" being initiated/enabled to use in this service.

But in this case its mainly to prevent "re-initialization", so that name would be better.